cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -957,7 +936,7 @@ public void getWorkflowssByApp(HttpRequest request, HttpResponder responder,
   @GET
   @Path("/streams")
   public void getStreams(HttpRequest request, HttpResponder responder) {
-    dataList(request, responder, Data.STREAM, null, null);
+    appFabricStreamHttpHandler.getStreams(rewriteRequest(request), responder, Constants.DEFAULT_NAMESPACE);
```

Comment:
I've just created a ``RESTMigrationUtils`` with the ``rewriteRequest`` method. Should be merged by afternoon. Please use it instead of this method. Not sure if you want to remove the rewriteRequest method from AbstractAppFabricHttpHandler in this PR, but lets not use it for newer code.